models/smtp.go

```diff
@@ -203,6 +204,12 @@ func PutSMTP(s *SMTP) error {
 		log.Error(err)
 		return err
 	}
+
+	record, err := GetSMTPByName(s.Name, s.UserId)
```

Comment:
As mentioned before, let's remove these checks that aren't necessary for these changes.